html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-001.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>HTML: window.open `features`: tokenization -- `noopener`</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#apis-for-creating-and-navigating-browsing-contexts-by-name">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+var windowURL = 'resources/close-self.html';
+
+test (t => {
+  var featureVariants = [
+    'NOOPENER',
+    'noOpenER'
+  ];
+  featureVariants.forEach(feature => {
+    var win = window.open(windowURL, '', feature);
+    assert_equals(win, null, `"${feature}" should be tokenized -> name "noopener"`);
+  });
+}, 'feature name should be converted to ASCII lowercase');
+
+test (t => {
+  var featureVariants = [
+    '  noopener',
+    'noopener = ',
+    ' no opener',
+    ', noopener',
+    '\tnoopener',
+    'no\nopener'
+  ];
+  featureVariants.forEach(feature => {
+    var win = window.open(windowURL, '', feature);
+    assert_equals(win, null, `"${feature}" should be tokenized -> name "noopener"`);
+  });
+}, 'tokenization should skip (ignore) ASCII whitespace');
+
+test (t => {
+  // Collect a sequence of code points that are not ASCII whitespace nor U+003D (=)
+  var featureVariants = [
+    'noopener=',
+    ',noopener=,',
+    'foo=,noopener=='
+  ];
+  featureVariants.forEach(feature => {
+    var win = window.open(windowURL, '', feature);
+    assert_equals(win, null, `"${feature}" should be tokenized -> name "noopener"`);
+  });
+}, 'tokenization should not collect "=" in feature name');
+
+test (t => {
+  var featureVariants = [
+    'noopener=false',
+    ',noopener=0, ',
```

Comment:
I was thinking a bit about this case the other day. Maybe we should specify `noopener` to use the same logic as other "boolean" features like `toolbar`. They're set to true if the value is empty string or "yes" or parses to a non-zero integer (not sure about the range, I think `short` in chromium), otherwise false.